nix/scripts/build.sh

```diff
@@ -44,6 +44,11 @@ if [[ -z "${TARGET}" ]]; then
   exit 1
 fi
 
+if [[ "${TARGET}" =~ ^(targets.status-go.mobile.android|targets.mobile.android.release)$ ]]; then
```

Comment:
We could benefit from a comment here and in `shell.sh` referencing the reason for this.